src/main/scala/mesosphere/marathon/core/group/GroupManager.scala

```diff
@@ -21,9 +21,21 @@ import scala.collection.immutable.Seq
   */
 trait GroupManager {
 
+  /**
+    * Get a root group, fetching it from a persistence store if necessary.
+    *
+    * @return a root group
+    */
   def rootGroup(): RootGroup
 
   /**
+    * Get a root group.
+    *
+    * @return a root group if it has been fetched already, otherwise [[None]]
+    */
+  def rootGroupOption(): Option[RootGroup]
```

Comment:
how about `findRootGroup`?

Reply:
I named it in line with the Scala standard library. Like `head` and `headOption`. What do you think?

Reply:
It's like this in the original patch so let's not change it in the cherry-pick (unless it's a bug)